horizon/templates/local_settings.py.erb

```diff
@@ -261,7 +261,7 @@ API_RESULT_PAGE_SIZE = 20
 
 # The timezone of the server. This should correspond with the timezone
 # of your entire OpenStack installation, and hopefully be in UTC.
-TIME_ZONE = "UTC"
```

Comment:
I can't find a definitive source to back this up, but I believe the point of  the `America/New_York` string instead of something like `EST` or `EDT` means that Daylight Savings Time should automatically be taken into account.